wlms/client.go

```diff
@@ -41,6 +41,13 @@ const (
 	RECENTLY_DISCONNECTED
 )
 
+// The protocol versions supported by the metaserver
+const (
+	BUILD19 int = 0
+	BUILD20     = 5
+	// TRUNK = 5
```

Comment:
This comment can go now?

Reply:
I left it in on purpose since it might be an entry in the future. But yes, it can go.

Reply:
We'd probably want to call id "BUILD21" or whichever build it is anyway, so I am in favor of getting rid of it.